### ISSUE TITLE
Hi，I would like to ask about the details of training and testing.

### ISSUE THREAD

**Issue by Yu-Wang-0801** [ZZUTK/SRNTT#3]
How to train and test without reference images during training and testing? When using only the reconstruction loss test, do you need to rename srntt.npz generated using only reconstruction loss training to srntt_init.npz for testing?

**Comment by ZZUTK** [comment]
For testing without ref, you can just set the parameter "ref_dir" to be None. If you want to train with reconstruction loss only, you can set the parameter "num_init_epochs" and leave "num_epochs" to be zero. During testing, set "use_init_model_only" to be True, which mean only load the init model, which is trained with reconstruction loss only.

**Comment by 949374585wangyu** [comment]
Hello, thank you very much for your reply! For the test without ref, after setting the parameter "ref_dir" to None, the program will report an error (ERROR Unrecognized ref_dir type!). Which positions do I need to modify? There is a code based on the pre-training model in the train.sh code. Is this based on the above code to train? Only num_epochs can be set in it, and the num_init_epochs number has no effect.

**Comment by Yu-Wang-0801** [comment]
Hello, thank you very much for your reply!  There is a code based on the pre-training model in the train.sh code. Is this based on the above code to train? Only num_epochs can be set in it, and the num_init_epochs number has no effect.
![image](https://user-images.githubusercontent.com/49390007/55781932-d176b200-5add-11e9-9b4c-dcefc635c200.png)

**Comment by ZZUTK** [comment]
The error comes from a typo "logging,info" on line 1277. Please change the
comma to dot.

If you want to train based on the init model (the model only trained by the
reconstruction loss), please the following:

python main.py \
--is_train True \
--input_dir data/train/${training_set}/input \
--ref_dir data/train/${training_set}/ref \
--map_dir data/train/${training_set}/map_321 \
--use_pretrained_model Ture \
--num_init_epochs 10 \
--num_epochs 0 \
--save_dir demo_training_srntt \ --use_init_model_only True

**Comment by Yu-Wang-0801** [comment]
Thank you very much for your reply. This time I bothered you. I really feel sorry. I am a beginner, so I still have a little question to ask you. I found that when I set use_pretrained_model to True, no matter how many num_init_epochs are set, it won't start training, only the number of num_epochs.
![image](https://user-images.githubusercontent.com/49390007/55863070-62b45a00-5bac-11e9-8345-d6b5f8bb3d31.png)

 Trained by the num_epochs iteration number, srntt.npz and discrim.npz will be updated continuously, and srntt_init.npz will not change. At this time, the test is still using srntt.npz to test, so I changed the training parameters. Look at it and see if it is wrong. First train the above part A (that is, train from all losses), and then train the following part B (that is, training only by reconstruction loss), do you see the order is wrong?
![image](https://user-images.githubusercontent.com/49390007/55863318-e2dabf80-5bac-11e9-8282-09f670dcb151.png)

 Another question is whether the code you wrote in the test code is tested one by one. Can you test multiple images at once?
![image](https://user-images.githubusercontent.com/49390007/55863327-e79f7380-5bac-11e9-9409-5b25360469b7.png)

**Comment by ximu1211** [comment]
> multiple images

Hello,have you solved multiple images testing problem?
